make gc docs referencable by lakefs cloud docs (#3491)
treeverse/lakeFS: docs/reference/garbage-collection.md

lakectl gc set-config lakefs://example-repo -f example_repo_gc_rules.json 
```

### From the lakeFS UI
{: .no_toc }

> 1. Click on the repository you would like to configure GC rules for
> 2. Go to _Settings_ -> _Retention_
> 3. Click _Edit policy_ and paste your GC rule into the text box as Json
> 4. Save you changes

Reviewer: ```suggestion
1. Naivgate to the main page of your repository.
2. Go to _Settings_ -> _Retention_.
3. Click _Edit policy_ and paste your GC rule into the text box as a JSON.
4. Save your changes.
```

Author: done thanks
